tutorials/power_distribution/power_distribution.c

```diff
@@ -0,0 +1,21 @@
+#include "tutorials/power_distribution/api.h"
+
+void power_device(gpio_t device) {
+    gpio_set_pin(device);
+}
+
+void power_all() {
```

Comment:
Technically this should also have `void` as the argument. Same for `power_off_all`.